components/raftstore-v2/src/raft/peer.rs

```diff
@@ -334,6 +344,17 @@ impl<EK: KvEngine, ER: RaftEngine> Peer<EK, ER> {
         mem::take(&mut self.has_ready)
     }
 
+    #[inline]
+    pub fn set_has_extra_write(&mut self) {
+        self.set_has_ready();
+        self.has_extra_write = true;
+    }
+
+    #[inline]
+    pub fn reset_has_extra_write(&mut self) -> bool {
```

Comment:
The semantics is different from `reset_flush_state`.
```suggestion
    pub fn take_has_extra_write(&mut self) -> bool {
```

Reply:
Now I change the implementation of `reset_flush_state`.